toolkit/tests/api/test_permissions.py

```diff
@@ -0,0 +1,88 @@
+from unittest.mock import MagicMock
+
+from pythonit_toolkit.api.permissions import IsService
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_service_to_service_token
+from ward import raises, test
+
+
+@test("Allowed callers pass the permission")
+async def _():
+    test_token = generate_service_to_service_token("test", "gateway", "users-backend")
+    PermissionClass = IsService(["gateway"], "test", "users-backend")
```

Comment:
can you separate the Arrange-Act-Assert phases with a blank line?

Reply:
cleaned up the tests a bit, let me know